README.md

```diff
@@ -278,6 +278,7 @@ Additional lists you might find useful:
 ## ORM and Datamapping
 *Plugins that implement object-relational mapping or data-mapping techniques.*
 
+- [Access JSON fields inner data from ORM](https://github.com/liqueurdetoile/cakephp-orm-json) - Behavior and Trait for selecting, finding, getting and setting properties and values inside JSON type fields through CakePHP ORM
```

Comment:
Please see the contribution guidelines. Several violations here.
One being the name of the plugin, more like [Lqdt/Coj plugin] probably
I do not like this nonspeaking name however, as it does not contain the intend inside.
VendorName/Purpose usually is better, in your case VendorName/OrmJson for example.